app/views/users/show.html.erb

```diff
@@ -1 +1,28 @@
-<h1>Here is the details of a given user</h1>
+<section class = "user-profile">
+    <section class = "user-card">
+      <div class="user-img">
+        <img src="<%= @users.photo %>" class="user-photo">
+      </div>
+      <div class="user-info">
+        <h1><%= @users.name %></h1>
+        <h2 class="post-counter"> Number of posts: <%= @users.posts_counter%></h2>
+      </div>
+    </section>
+    
+    <section class = "user-bio">
+      <h5>Bio</h5>
+      <%= @users.bio %>
+    </section>
+    <section class = "user-posts">
+      <% @users.posts.each do |post| %>
+        <div class= "post">
+          <h2><%= post.title %></h2>
+          <p><%= post.text %></p>
+          <div>
+            <p>Comments: <%= post.comments_counter %></p>
+            <p>Likes: <%= post.likes_counter %></p>
+        </div>
+      <% end %>
+    </div>
+    <%= link_to "See all posts", user_posts_path(user_id: @users.id), class: "all-post" %>
+</section>
```

Comment:
- [ ] [REQUIRED] You did a great job here @Wineshuga , however, a part of the project requirement states that you should  use `recent_posts` method to render the user's 3 most recent posts. Take a look at the images below. Kindly fix this.

This method:
https://github.com/Wineshuga/pen-it/blob/a79f42b6cc38923e429a00fa36bc3dc076adf5f0/app/models/user.rb#L9

Your Design
<img width="1251" alt="Screenshot 2023-11-03 at 16 32 06" src="https://github.com/Wineshuga/pen-it/assets/58692590/b3bcb203-0823-48f7-86c5-887dbf2e803e">

Required Design
<img width="470" alt="Screenshot 2023-11-03 at 16 31 19" src="https://github.com/Wineshuga/pen-it/assets/58692590/401af676-69ac-44a1-bdb3-d2f890c3f784">